src/garage/tf/regressors/categorical_mlp_regressor.py

```diff
@@ -274,7 +218,7 @@ def vectorized(self):
     @property
     def distribution(self):
         """garage.tf.distributions.DiagonalGaussian: Distribution."""
-        return self._dist
+        return self._network.dist
```

Comment:
This name should be consistent with the one you specified inside the model (in this case `categorical_dist`).
I suggest renaming the name in `network_output_spec` to just `dist` since most of the models use `dist` as well.

Is there a test that will make sure `self._network.dist` exists?

Reply:
Got it. Will do.
I will add the test to test `self._network.dist`.